src/main/src/jetty/override-web.xml

```diff
@@ -31,4 +31,10 @@
        <res-type>javax.sql.DataSource</res-type>
        <res-auth>Container</res-auth>
     </resource-ref>
+
+    <resource-ref>
+        <res-ref-name>jdbc/harvest</res-ref-name>
```

Comment:
Harvest is the name of our database and jndi entry. Seems too specific to include in a test configuration.